src/scenarios/server/sse-polling.ts

```diff
@@ -0,0 +1,288 @@
+/**
+ * SSE Polling conformance test scenarios for MCP servers (SEP-1699)
+ *
+ * Tests that servers properly implement SSE polling behavior including:
+ * - Sending priming events with event ID and empty data
+ * - Sending retry field before closing connection
+ */
+
+import { ClientScenario, ConformanceCheck } from '../../types.js';
+import { EventSourceParserStream } from 'eventsource-parser/stream';
+
+export class ServerSSEPollingScenario implements ClientScenario {
+  name = 'server-sse-polling';
+  description =
+    'Test server sends SSE priming event and retry field (SEP-1699)';
+
+  async run(serverUrl: string): Promise<ConformanceCheck[]> {
+    const checks: ConformanceCheck[] = [];
+
+    try {
+      // Make a GET request to establish SSE stream
+      const response = await fetch(serverUrl, {
+        method: 'GET',
+        headers: {
+          Accept: 'text/event-stream'
+        }
+      });
+
+      // Check if server supports SSE GET endpoint
+      if (response.status === 405) {
+        checks.push({
+          id: 'server-sse-polling-endpoint',
+          name: 'ServerSSEPollingEndpoint',
+          description: 'Server supports SSE GET endpoint',
+          status: 'INFO',
+          timestamp: new Date().toISOString(),
+          specReferences: [
+            {
+              id: 'SEP-1699',
+              url: 'https://github.com/modelcontextprotocol/modelcontextprotocol/issues/1699'
+            }
+          ],
+          details: {
+            serverUrl,
+            statusCode: response.status,
+            message:
+              'Server does not support SSE GET endpoint (405 Method Not Allowed)'
+          }
+        });
+        return checks;
+      }
+
+      // Server may require session context for standalone SSE GET
+      if (response.status === 400) {
+        checks.push({
+          id: 'server-sse-polling-endpoint',
+          name: 'ServerSSEPollingEndpoint',
+          description: 'Server supports SSE GET endpoint',
+          status: 'INFO',
+          timestamp: new Date().toISOString(),
+          specReferences: [
+            {
+              id: 'SEP-1699',
+              url: 'https://github.com/modelcontextprotocol/modelcontextprotocol/issues/1699'
+            }
+          ],
+          details: {
+            serverUrl,
+            statusCode: response.status,
+            message:
+              'Server requires session context for standalone SSE GET endpoint (400 Bad Request)'
+          }
+        });
+        return checks;
+      }
+
+      if (!response.ok) {
+        checks.push({
+          id: 'server-sse-polling-connection',
+          name: 'ServerSSEPollingConnection',
+          description: 'Server accepts SSE GET request',
```

Comment:
See comment above, I think this should either apply to both GET and POST or even specifically to POST

Reply:
Yes, will fix.